commitserver/commit/commit.go

```diff
@@ -106,6 +108,18 @@ func (s *Service) handleCommitRequest(ctx context.Context, logCtx *log.Entry, r
 		return out, "", fmt.Errorf("failed to checkout target branch: %w", err)
 	}
 
+	logCtx.Debug("Checking out if hydrated already")
+	metadataPath := path.Join(dirPath, "hydrator.metadata")
+	fileData, err := os.ReadFile(metadataPath)
+	if err == nil {
+		var hydratorMetadata hydratorMetadataFile
+		json.Unmarshal(fileData, &hydratorMetadata)
+		if hydratorMetadata.DrySHA == r.DrySha {
+			logCtx.Debug("Already Hydrated")
```

Comment:
Maybe include the sha in the debug message.

Reply:
that's a good idea! I applied it